contracts/RMRK/extension/erc20Holder/ERC20Holder.sol

```diff
@@ -19,32 +19,42 @@ abstract contract ERC20Holder is IERC20Holder {
         address erc20Contract,
         uint256 tokenId
     ) external view returns (uint256) {
-        return balances[tokenId][erc20Contract];
+        return _balances[tokenId][erc20Contract];
     }
 
     /**
-     * @inheritdoc IERC20Holder
+     * @notice Transfer ERC-20 tokens to a specific token
+     * @dev The ERC-20 contract must have approved this contract to transfer the ERC-20 tokens
+     * @dev The balance MUST be transferred from the message sender
```

Comment:
I think that the balance is transferred from the smart contract as the tokens are owned by the NFT, right?

```suggestion
     * @dev The balance MUST be transferred from this smart contract.
```

Reply:
Nope, same as previous 2 comments